From a9e24191c4c72431e87ff3b2f439fe5db2d8f05c Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Wed, 19 Jan 2022 11:28:21 +0200
Subject: [PATCH] [SW-58071] package: arcboot: copy die1 image for rom

This patch add support to copy die1 image of arcboot ROM to images
folder for gaudi3.

Change-Id: I09f999a185f5362f4e29209f8dda6513321150cf
---

diff --git a/package/arcboot/arcboot.mk b/package/arcboot/arcboot.mk
index 5dfe13f..c9e8b24 100644
--- a/package/arcboot/arcboot.mk
+++ b/package/arcboot/arcboot.mk
@@ -14,6 +14,7 @@
 define ARCBOOT_INSTALL_IMAGES_CMDS
 	if [ "$(HL_DEV)" == "gaudi3_arc" ]; then \
 		$(INSTALL) -D -m 0644 $(@D)/arcboot_rom.bin $(BINARIES_DIR)/arcboot_rom_d0.bin; \
+		$(INSTALL) -D -m 0644 $(@D)/arcboot_rom_d1.bin $(BINARIES_DIR)/arcboot_rom_d1.bin; \
 	else \
 		$(INSTALL) -D -m 0644 $(@D)/arcboot_ram.bin $(BINARIES_DIR)/arcboot_ram.bin; \
 		$(INSTALL) -D -m 0644 $(@D)/arcboot_rom.bin $(BINARIES_DIR)/arcboot_rom.bin; \
